www/pad/inner.js

```diff
@@ -757,6 +757,7 @@ define([
         */
 
         framework.onEditableChange(function(unlocked) {
+            $('#cp-app-pad-editor').show();
```

Comment:
This will break the "cache" system. While we wait for the content to be editable (meaning it is fully synced with the server), we display a read-only "cached" version of the document. This won't trigger en "editable change" event and will result in a page showing only the toolbar.